test/api/liskApi.js

```diff
@@ -1,28 +1,24 @@
-if (typeof module !== 'undefined' && module.exports) {
```

Comment:
This part was especially to have the tests also running in a browser. Did you verify if tests in the browser still work? I cannot test right now because the grunt is not working as you mentioned. You can find those tests in browsertest/. It would be good to maintain it because lisk-js is expected to run in nodejs and the browser